conjure-java-core/src/test/java/com/palantir/conjure/java/AsyncRequestProcessingTest.java

```diff
@@ -162,6 +162,14 @@ public void testTimeout() throws IOException {
         }
     }
 
+    @Test
+    public void testFiveSecondTimeoutIsNotReached() throws IOException {
+        try (Response response = requestToDelayEndpointWithFiveSecondTimeout(OptionalInt.of(2000))) {
+            assertThat(response).matches(Response::isSuccessful);
+            assertThat(response.body().string()).isEqualTo("\"Completed after 2000ms\"");
```

Comment:
Is there a similar test to determine if the timeout is enforced (I didn't see it here)?

Reply:
The test above verified the timeout works -- this test verified the same thing, but overrides with a longer timeout to ensure it can be increased